vector/v.surf.bspline/v.surf.bspline.html

```diff
@@ -2,7 +2,7 @@ <h2>DESCRIPTION</h2>
 
 <em>v.surf.bspline</em> performs a bilinear/bicubic spline
 interpolation with Tykhonov regularization. The <b>input</b> is a 2D
-or 3D vector <em>points</em> map. Values to interpolate can be the z
+or 3D vector <em>point</em> map. Values to interpolate can be the z
```

Comment:
Good point. I do no think so. I have thought about this while reviewing this manual. I believe everything could be better be renamed to "point layer". Will make an amend to the PR on that.

Reply:
> Does point or points have a special meaning in grass context?

Looong time ago it used to be a special format ("sites"), which was later integrated into the vector format. So, the wording may be streamlined.

Reply:
I believe it will be better if we use "point vector layer" instead of "vector points map". The original file uses the word "map" where "layer" is perhaps more appropriate. Map is not incorrect,perhaps just inconsistent with the other manuals or GIS in general?

Reply:
Thanks for the historical context, I didn't stumble upon "sites" yet in my time here.